redirector/main.go

```diff
@@ -176,10 +181,15 @@ func (r *root) findKBFSMount(ctx context.Context) (
 		// "/home/user/keybase", and make sure it doesn't match mounts for
 		// another run mode, say "/home/user/keybase.staging".
 		i := strings.Index(mp, r.runmodeStr)
+		str := r.runmodeStr
+		if i < 0 {
+			i = strings.Index(mp, r.runmodeStrFancy)
+			str = r.runmodeStrFancy
+		}
 		if i < 0 {
 			continue
 		}
-		if len(mp) > i+len(r.runmodeStr) && mp[i+len(r.runmodeStr)] != '/' {
+		if len(mp) > i+len(str) && mp[i+len(str)] != '/' {
```

Comment:
The second condition is specifically for `/home/user/.local/share/keybase/fs` I suppose?

Reply:
Yes, or just generally to be flexible.